services/ui-src/src/components/cards/TemplateCard.tsx

```diff
@@ -44,8 +47,12 @@ export const TemplateCard = ({
         )}
         <Flex sx={sx.cardContentFlex}>
           <Heading sx={sx.cardTitleText}>{verbiage.title}</Heading>
-          <Text>{cardText}</Text>
-
+          {/* TODO: Remove LD flag once MLR is released */}
+          {!mlrReport && templateName === "MLR" ? (
+            <Text>{verbiage.body.unavailable}</Text>
+          ) : (
+            <Text>{cardText}</Text>
```

Comment:
Thought: It seems like we're splitting the logic kind of oddly here. `cardText` is set above and considers if the verbiage link is available or not. Maybe we should make a new variable on line 34 that does the check currently on line 51, then combine it with verbiage.link check on 35 to get the cardText. That new variable could be referenced elsewhere and cleans this up a bit.

Reply:
The const we set at the top could be an object, like: 
```
const enabledReports = {
  MCPAR: // logic for MCPAR enabled here,
  MLR: // logic for MLR enabled here, which is likely just mlrReportEnabled,
}
```

then the check for each one below is no longer `templateName === "MLR"`, it becomes: 
`enabledReports[templateName]` -- and if that's `true`, you're good to go.

Reply:
Happy to walk through any/all of this. 👍